bin/Makefile.am

```diff
@@ -22,15 +22,11 @@ MAINTAINERCLEANFILES = \
 	$(NULL)
 
 EXTRA_DIST = \
-	ovirt-ha-agent.in \
```

Comment:
https://www.gnu.org/software/automake/manual/html_node/Basics-of-Distribution.html

Means this makes sure files are added in the tar when doing 'make dist'
But as the source is not modified anymore, I THINK you can remove it here.

Reply:
You can remove the whole EXTRA_DIST variable here